tap_google_ads/streams.py

```diff
@@ -141,12 +150,14 @@ def on_giveup_func(err):
 
 @backoff.on_exception(backoff.expo,
                       (GoogleAdsException,
+                       ServerError, TooManyRequests,
```

Comment:
Super minor, but since we seem to have all the others on their own line, what would you think about splitting this one up as well?

Reply:
We have separated out exceptions on separate lines just for better readability. There is no other reason for that.